src/Plugin/Derivative/ViewsEntityRowDynamicFormat.php

```diff
@@ -22,9 +22,9 @@ class ViewsEntityRowDynamicFormat extends ViewsEntityRow {
    */
```

Comment:
On Line no-6 , Drupal\Core\Entity\EntityManagerInterface 
https://api.drupal.org/api/drupal/core%21lib%21Drupal%21Core%21Entity%21EntityManagerInterface.php/interface/EntityManagerInterface/8.8.x

Reply:
Fixed in lasted commit.